packages/twenty-chrome-extension/src/db/person.ts

```diff
@@ -0,0 +1,50 @@
+import { Person } from '~/db/types/person';
+
+import handleQueryParams from '../utils/handleQueryParams';
+import requestDb from '../utils/requestDb';
+
+export const fetchPerson = async (personData: Person) => {
+  const query = `
+    query FindManyPeople {
+      people {
+        edges {
+          node {
+            name {
+              firstName
+              lastName
+            }
+          }
+        }
+      }
+    }
+  `;
+  const res = await requestDb(query);
+  if (res.errors) {
+    throw Error(res.Errors);
+  }
+  if (res.data) {
+    const person: Person[] = res.data.people.edges.filter(
+      (edge: any) =>
+        edge.node.name.firstName === personData.name.firstName &&
```

Comment:
This is not the right way to do it - you should do it on the backend not on the frontend :)

You can probably leverage the duplicate endpoint that was recently introduced!

Reply:
There are many reasons why it shouldn't be done on the frontend (e.g. performance), but primary reason is that there is pagination so this just won't work if you have 200 contacts on your workspace